android/src/main/kotlin/com/chargebee/flutter/sdk/ChargebeeFlutterSdkPlugin.kt

```diff
@@ -309,10 +309,29 @@ class ChargebeeFlutterSdkPlugin : FlutterPlugin, MethodCallHandler, ActivityAwar
 fun CBProduct.toMap(): Map<String, Any> {
     return mapOf(
         "productId" to productId,
-        "productPrice" to productPrice,
+        "productPrice" to skuDetails.priceAmountMicros / 1_000_000.0,
```

Comment:
Let's extract it to a variable/method to make it explicit why we are doing this. 
Ex. 
```
fun convertPriceAmountInMicros(priceAmountMicros: long): double {
  return priceAmountMicros/1_000_000.0
}
```